Base/usr/share/man/man1/syscall.md

```diff
@@ -0,0 +1,58 @@
+## Name
+
+syscall - test a system call
+
+## Synopsis
+
+`syscall [-o] [-l] [-h] entry [args; buf==BUFSIZ buffer]`
+
+## Description
+
+syscall(1) is an effort to port the plan9 syscall command to Serenity. It can invoke a system call with the given arguments from the command line.
+
+## Options
+
+* `-o`: Output to stdout the contents of the buffer argument specified as buf.
+* `-l`: Print all the serenity system calls and exit. Note that not all the system calls can be invoked using this tool.
+* `-h`: Print a help message and exit.
+
+## Examples
+
+Write a string to standard output:
+
+```sh
+syscall write 1 hello 5
+```
+
+Read a string from the standard input and output the buffer contents to stdout:
+
+```sh
+syscall -o read 0 buf 3
+```
+
+Get the pid of the current running process:
+
+```sh
+syscall getpid
+```
+
+Sleep for 3 seconds:
+
+```sh
+syscall sleep 3
+```
+
+Create a directory:
+
+```sh
+syscall mkdir my-dir 755
```

Comment:
Will "775" actually get parsed as octal?

Reply:
I don't think so, I've changed it to 0755 and it's working as expected following the chmod convention.